docs/examples/json_debbuging_reddit.py

```diff
@@ -0,0 +1,35 @@
+import json
```

Comment:
This examples directory is intended to contain completed scripts that are part of the documentation. Unless you want to include a tutorial on using this feature, I think a simple example in the code comments should suffice.